crates/re_types/definitions/rerun/components/material.fbs

```diff
@@ -9,7 +9,7 @@ namespace rerun.components;
 
 // ---
 
-/// Material properties of a mesh.
+/// Material properties of a mesh, e.g. its color multiplier.
```

Comment:
bit of a strange example, but given that we don't have the texture in there unfortunately, it's rather accurate